glam/api/views.py

```diff
@@ -451,9 +451,7 @@ def _log_probe_query(request):
     UsageInstrumentation(
         action_type=UsageInstrumentation.ACTION_PROBE_SEARCH,
         context=query,
-        tracking_id=request.COOKIES.get(
-            "session", ""
-        ),  # FIXME this field isn't always present. We need to find a better replacement to allow for accurate analysis
+        tracking_id="",  # FIXME use something that allows for accurate analysis
```

Comment:
We should look into using Django session middleware: https://docs.djangoproject.com/en/4.0/topics/http/sessions/

MDN quick tutorial: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Sessions